sources/src/fsdb_unix.c

```diff
@@ -14,6 +14,10 @@
 #include "fsdb.h"
 #include "misc.h"
 
+#ifdef __CELLOS_LV2__
```

Comment:
Does not really matter, but would be cleaner if this was inside __LIBRETRO__ below.

Reply:
Done